ourjapanlife/findadoc-server
# improve prisma migrate to add prod deploy

# What changed
- I noticed we were getting an error in the docker logs recommending using `prisma migrate deploy` instead of `prisma migrate dev`

# Testing instructions
Try `docker compose up` and the error should be gone. Ensure the seeding completes by reading the log messages: `🌱  The seed command has been executed.`
